Entity/Repositories/SentEmailRepository.php

```diff
@@ -12,4 +13,42 @@
  */
 class SentEmailRepository extends EntityRepository
 {
+    public function search($searchParams = [])
+    {
+        $queryBuilder = $this->createQueryBuilder('e');
+
+        if (empty($searchParams)) {
+            return $queryBuilder;
+        }
+
+        $searchAttributes = [
+            'recipients',
+            'template',
+            'sent',
+            'variables',
+            'token'
+        ];
+
+        foreach ($searchAttributes as $attribute) {
+            if (empty($searchParams[$attribute])) {
+                continue;
+            }
+
+            $attributeValue = $searchParams[$attribute];
+
+            $queryBuilder->andWhere('e.'.$attribute.' LIKE :'.$attribute)
+                ->setParameter(':'.$attribute, '%'.$attributeValue.'%');
```

Comment:
it does not throw an error, but there is not supposed to be a ":" at the beginning of the key-name.
`->setParameter($attribute, '%'.$attributeValue.'%');`